### PR TITLE
Fix border hover cosmetic issues

### PR DESCRIPTION
**Pull request by ddc22**
#### Changes proposed in this Pull Request

* Fixed page selector border issues
image

hover

Refactored Entire Component
hellox

#### Testing instructions
Go  through `start/do-it-for-me`
On the mage selector make sure the hover effect matches the above left design.

Related to #

### CODE REVIEW

File: client/signup/difm/components/BrowserView.tsx [hunk 1]
```diff
@@ -15,31 +15,46 @@ import homePage from 'calypso/signup/difm/images/home-page.svg';
 import photoGallery from 'calypso/signup/difm/images/photo-gallery.svg';
 import serviceShowcase from 'calypso/signup/difm/images/service-showcase.svg';
 
-const Container = styled.div< { isSelected?: boolean; isClickDisabled?: boolean } >`
-	border: 2px solid
+const ContainerBoarder = styled.div< { isSelected?: boolean; isClickDisabled?: boolean } >`
```
Comment (aneeshd16) [inline client/signup/difm/components/BrowserView.tsx]: Typo. :) Should be `ContainerBorder`

**Review by aneeshd16 — COMMENTED** [review]
There is some overlap happening at the corners here, I haven't pinpointed the cause yet:
image
image

File: client/signup/difm/components/BrowserView.tsx (hunk 1 above)
Comment (ddc22) [reply client/signup/difm/components/BrowserView.tsx]: Lol thanks for that 😅

File: client/signup/difm/components/BrowserView.tsx [hunk 2]
```diff
@@ -8,66 +8,94 @@ import {
...
+    top: 12px;
+	&:hover{
+		width: ${ ( { isSelected } ) => ( isSelected ? '224px' : '221px' ) };
+	}
+}`;
+
+function Header( props: { isSelected?: boolean; isClickDisabled?: boolean } ) {
+	return (
+		<HeaderContainer { ...props }>
+			<img src={ threeDots } alt="page preview" />
```
Comment (ddc22) [inline client/signup/difm/components/BrowserView.tsx]:
```suggestion
			<img src={ threeDots } alt="three dots" />
```

File: client/signup/difm/components/BrowserView.tsx [hunk 3]
```diff
@@ -8,66 +8,94 @@ import {
...
+import contactPage from 'calypso/signup/difm/images/page-descriptions/contact-page.svg';
+import homePage from 'calypso/signup/difm/images/page-descriptions/home-page.svg';
+import photoGallery from 'calypso/signup/difm/images/page-descriptions/photo-gallery.svg';
+import serviceShowcase from 'calypso/signup/difm/images/page-descriptions/service-showcase.svg';
+import threeDots from 'calypso/signup/difm/images/three-dots.svg';
+
+const ContainerShadow = styled.div< { isSelected?: boolean; isClickDisabled?: boolean } >`
+	transition: box-shadow 400ms ease-in-out, border-box 400ms ease-in-out;
+	&:hover {
+	}
```
Comment (aneeshd16) [inline client/signup/difm/components/BrowserView.tsx]: Looks like you forgot to remove this. :)

File: client/signup/difm/components/BrowserView.tsx [hunk 4]
```diff
@@ -8,66 +8,94 @@ import {
...
-import serviceShowcase from 'calypso/signup/difm/images/service-showcase.svg';
+import aboutPage from 'calypso/signup/difm/images/page-descriptions/about-page.svg';
+import blogPage from 'calypso/signup/difm/images/page-descriptions/blog-page.svg';
+import contactPage from 'calypso/signup/difm/images/page-descriptions/contact-page.svg';
+import homePage from 'calypso/signup/difm/images/page-descriptions/home-page.svg';
+import photoGallery from 'calypso/signup/difm/images/page-descriptions/photo-gallery.svg';
+import serviceShowcase from 'calypso/signup/difm/images/page-descriptions/service-showcase.svg';
+import threeDots from 'calypso/signup/difm/images/three-dots.svg';
+
+const ContainerShadow = styled.div< { isSelected?: boolean; isClickDisabled?: boolean } >`
```
Comment (aneeshd16) [inline client/signup/difm/components/BrowserView.tsx]: Can these props can be removed since they are not being used?

**Review by aneeshd16 — COMMENTED** [review]
Works perfectly! I left some minor code feedback.

File: client/signup/difm/components/BrowserView.tsx (hunk 4 above)
Comment (ddc22) [reply client/signup/difm/components/BrowserView.tsx]: I actually removed this whole container, which is unnecessary :)

**Comment by ddc22** [comment]
I have fix the issues you raised @aneeshd16  :)

**Review by aneeshd16 — APPROVED** [review]

**Comment by aneeshd16** [comment]
LGTM, I'll ship this. 👍

**Pull request merged**